WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -284,3 +284,24 @@ $(function() {
     $('[data-toggle="tooltip"]').tooltip();
   });
 });
+
+// Helpers
+
+// Executes the given function on the specified page/pages.
+// The given string should have a format: '<controller>#<action>'.
+// The action name preceded by '#' is optional.
+// Could be followed by comma and another pair.
+// Example: 'users#show, users#edit, users#update, competitions'.
+function onPage(controllersWithActions, fun) {
+  controllersWithActions = controllersWithActions.replace(/\s/g, '');
+  controllersWithActions = controllersWithActions.split(',');
```

Comment:
You might need to strip the whitespace after splitting at `,` for your example to work.